### ISSUE TITLE
visualize training data some problems

### ISSUE THREAD

**Issue by Gzzgz** [open-mmlab/mmhuman3d#121]
I converted surreal or h36m data for training. I want to make sure the label of one img is right. so I visualize them.
eg:
for keypoints3d:
<img width="871" alt="图片" src="https://user-images.githubusercontent.com/15968092/156338533-c833273c-7586-4cda-bd00-cb8dc3e7ba17.png">
this is a res from surreal data. the kpts3d positions misalign to the img. 
may be it wrong? What shall I do?

THX

**Comment by JasonBoy1** [comment]
@LazyBusyYang

**Comment by LazyBusyYang** [comment]
3D keypoints have a different coordinate from 2D image, which means that you cannot overlay locations of a 3d point on img.
If you have the correct camera parameter, here's a possible way.
1. Construct an instance of `CameraParameter` with K, R, T, H, W.
2. Call `camera_parameter.export_to_perspective_cameras()` to get a camera in pytorch3d.
3. Project kp3d back to 2d camera spcae. `keypoints2d = pytorch3d_perspective_camera.transform_points_screen(
            points=keypoints3d[..., :3])[..., :2]`
4. Visualize kp2d by `visualize_kp2d`, offering keypoints2d and images at the same time.
note: It may fail because of wrong cam parameters. If it happens, a simple try with inversed extrinsic might do.

**Comment by Gzzgz** [comment]
OK.THX
